lib/3scale/backend/configuration.rb

```diff
@@ -121,8 +121,8 @@ def parse_int(value, default)
         config.master.metrics.transactions_authorize = CONFIG_MASTER_METRICS_TRANSACTIONS_AUTHORIZE_DEFAULT
       end
 
-      if config.master_service_id.nil? || config.master_service_id.to_s.empty?
-        raise InvalidMasterServiceId
```

Comment:
doesn't this removal render this error completely unused?

Reply:
Good catch. Fixed :+1: